### DISCUSSION TITLE
DietPi-Software | Readarr

### DISCUSSION THREAD

**Discussion by cadric** [MichaIng/DietPi#5003] [Software Request] [answered]
Software title | Readarr
    Short description | Readarr is an ebook and audiobook collection manager for Usenet and BitTorrent users.
    Official URL (if available) | https://readarr.com/
    GitHub URL (if available) | https://github.com/Readarr/Readarr
    (Official) install documentation | https://readarr.com/#downloads-v1-linux-debianubuntu

Are there similar/alternative software titles available with DietPi-Software?

    Not that I am aware of.

What makes your requested software better than the above solutions, if available?

    There are no ebook or audiobook downloader available to dietpi atm

How can DietPi make the installation easier or compatible, than following the install instructions or do APT installation, if available?

    Everything is just easier when Dietpi does it for me :)

Can you provide the installation steps that you would suggest DietPi-Software to do?

    Nope.

Are you willing to help maintaining the software installation, e.g. in case of needed setup changes due to updates etc.? This is not needed, but could speed up our decision to implement it, as man power is always a topic 😉.

    Unfortunately not

**Comment by Joulinar** [comment, marked as answer]
Duplicate of #4246
